src/main/java/com/spotinst/sdkjava/model/ElastigroupConverter.java

```diff
@@ -1754,6 +1881,131 @@ private static ElastigroupCodeDeploy toBl(ApiCodeDeploy apiCodeDeploy) {
+                }
+            }
+            blBeanstalk = blBeanstalkBuilder.build();
+        }
+
+        return blBeanstalk;
+
+    }
+
+    private static ElastigroupDeploymentPreferences toBl(ApiElastigroupDeploymentPreferences apiDeploymentPreferences) {
+        ElastigroupDeploymentPreferences blDeploymentPreference = null;
+
+        if (apiDeploymentPreferences != null) {
+            ElastigroupDeploymentPreferences.Builder blDeploymentPreferenceBuilder = ElastigroupDeploymentPreferences.Builder.get();
+
+            if (apiDeploymentPreferences.isAutomaticRollSet()) {
+                blDeploymentPreferenceBuilder.setAutomaticRoll(apiDeploymentPreferences.getAutomaticRoll());
+            }
+
+            if (apiDeploymentPreferences.isBatchSizePercentageSet()) {
+                blDeploymentPreferenceBuilder.setBatchSizePercentage(apiDeploymentPreferences.getBatchSizePercentage());
+            }
+
+            if (apiDeploymentPreferences.isGracePeriodSet()) {
+                blDeploymentPreferenceBuilder.setGracePeriod(apiDeploymentPreferences.getGracePeriod());
+            }
+
+            if (apiDeploymentPreferences.isStrategySet()) {
+                if (apiDeploymentPreferences.getStrategy() != null) {
+                    blDeploymentPreferenceBuilder.setStrategy(toBl(apiDeploymentPreferences.getStrategy()));
+                }
+            }
+            blDeploymentPreference = blDeploymentPreferenceBuilder.build();
+        }
+
+        return blDeploymentPreference;
+
+    }
+
+    private static BeanstalkStrategy toBl(ApiBeanstalkStrategy apiStrategy) {
+        BeanstalkStrategy blStrategy = null;
+
+        if (apiStrategy != null) {
+            BeanstalkStrategy.Builder blStrategyBuilder = BeanstalkStrategy.Builder.get();
+
+            if (apiStrategy.isActionSet()) {
+                blStrategyBuilder.setAction(apiStrategy.getAction());
+            }
+
+            if (apiStrategy.isShouldDrainInstancesSet()) {
+                blStrategyBuilder.setShouldDrainInstances(apiStrategy.getShouldDrainInstances());
+            }
+            blStrategy = blStrategyBuilder.build();
+        }
+
+        return blStrategy;
+
+    }
+
+    private static ManagedActions toBl(ApiManagedActions apiManagedActions) {
+        ManagedActions blManagedActions = null;
+
+        if (apiManagedActions != null) {
+            ManagedActions.Builder blManagedActionsBuilder = ManagedActions.Builder.get();
+
+            if (apiManagedActions.isPlatformUpdateSet()) {
+                if (apiManagedActions.getPlatformUpdate() != null) {
+                    blManagedActionsBuilder.setPlatformUpdate(toBl(apiManagedActions.getPlatformUpdate()));
+                }
+            }
+            blManagedActions = blManagedActionsBuilder.build();
+        }
+
+        return blManagedActions;
+
+    }
+
+    private static BeanstalkPlatformUpdate toBl(ApiBeanstalkPlatformUpdate apiStrategy) {
```

Comment:
change variable name to apiPlatformUpdate

Reply:
Done